Makefile

```diff
@@ -501,11 +499,13 @@ update-dashboards:
 gomodtidy:
 	@printf $(COLOR) "go mod tidy..."
 	@go mod tidy
+	@go list -deps $(FUNCTIONAL_TEST_ROOT) | sort -u | grep '^[a-z]\+\.[a-z.]\+/' | grep -v go.temporal.io > develop/buildkite/third_party_deps.txt
```

Comment:
Why exclude the Go standard library packages; wouldn't we also benefit from pre-building them?